CMakeLists.txt

```diff
@@ -334,13 +334,76 @@ target_include_directories(
   dosbox PUBLIC include ${CMAKE_CURRENT_BINARY_DIR}/include
 )
 
-# TODO This happens on every execution, better to have a target with
-# dependencies instead
-add_custom_target(copy_assets ALL
-  COMMAND ${CMAKE_COMMAND} -E copy_directory
-    "${CMAKE_CURRENT_SOURCE_DIR}/contrib/resources"
-    "${CMAKE_CURRENT_BINARY_DIR}${RESOURCE_COPY_PATH}"
+# Add contrib directory to make the list of resource files available
+add_subdirectory(contrib)
```

Comment:
Yeah, so just copy the whole `contrib/resources/*` here wholesale as per my earlier comment. What-you-see-is-what-you-get (in the final release package) is a clear to model to follow with the `resources` dir.